binary-string-types.ipynb

```diff
@@ -0,0 +1,532 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Variable sized types\n",
+    "\n",
+    "## Variable length binary, large binary, string and large string\n",
```

Comment:
```suggestion
    "## Variable length binary and string\n",
```

(keeping the title a bit simpler here? It was also not yet complete because it didn't include the view types)